Kata_v1.0.0/src/kata_v2.js

```diff
@@ -0,0 +1,32 @@
+function comp(array1, array2) {
+    var comp1 = []; 
+    if( array1 == null || array2 == null || array1.length === 0 || array2.length === 0){
+        return false;
+    }else {
+        for (var j in array2) {
+            comp1.push(Math.sqrt(array2[j]));
+          }
+          let resultsActions = array1.map(d => d);
+          return comp1.every(i => resultsActions.includes(i));
+    }
+    
+  }
+  let resultsActions = array1.map((d) => d);
+  return comp1.every((i) => resultsActions.includes(i));
+}
+
```

Comment:
Line 14 - 16 should not be here. It will cause errors. 
Why did you rewrite this function?